.github/workflows/build-test.yml

```
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  workflow_dispatch: { }
+  workflow_call: { }
+
+jobs:
+  build-test-book:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        python-version: [ "3.9" ]
```

```suggestion
        python-version: [ "3.10" ]
```
We can go even with higher version, as most of the libs are dropping the support of 3.9